index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Unlimited Love Launch Party</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!-- nav mobile -->
+    <header>
+      <nav class="navbar">
+        <a href="#" class="toplogo">
+          <img src="./assets/Images/chililogo.png" alt="band logo"/>
+        </a>
+        <ul class="nav-menu">
```

Comment:
- [ ] Kindly add this section to your design on your home and about pages, you have to stick to the original layout.

Your design
![image](https://user-images.githubusercontent.com/35535130/163289802-66704112-0657-456e-9170-7de5230ea461.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/163289916-b855bad5-82d8-4609-a34a-d4316bd7f07b.png)

Reply:
Thank you for the comments, I just finished working on this change